tests/TestCase.php

```diff
@@ -43,20 +55,22 @@ protected function setUp(): void
             'views' => $this->sourcePath,
             'destination' => $this->destinationPath,
         ];
-        $this->filesystem = new Filesystem();
-        $this->tempPath = $this->app->cachePath();
-        $this->prepareTempDirectory();
+    }
+
+    protected function createTmp(): void
+    {
+        mkdir($this->tmp = __DIR__ . '/fixtures/tmp/' . static::haiku());
+
+        // TODO this creates Jigsaw's cache directory in the root of this repo
```

Comment:
I think this comment can be removed, right?

Reply:
I don't think so—that was already the behaviour before this PR and the Blade compiler relies on that directory existing. Eventually I want to fix it but that might require more fiddling with how Jigsaw determines its 'base path' and working directory.

But actually only 5 tests fail without it so let me see!